ide/che-core-ide-templates/src/main/resources/samples.json

```diff
@@ -1,5 +1,74 @@
 [
   {
+    "name": "web-java-petclinic",
+    "displayName": "web-java-petclinic",
+    "path": "/web-java-petclinic",
+    "description": "Petclinic app using MySQL as persistent storage",
+    "projectType": "maven",
+    "mixins": [],
+    "attributes": {
+      "language": [
+        "java"
+      ]
+    },
+    "modules": [],
+    "problems": [],
+    "source": {
+      "type": "git",
+      "location": "https://github.com/che-samples/web-java-spring-petclinic.git",
+      "parameters": {
+        "branch":"mysql"
+      }
+    },
+    "commands": [
+      {
+        "name": "build",
+        "type": "mvn",
+        "commandLine": "mvn -f ${current.project.path} clean install -DskipTests && cp ${current.project.path}/target/*.war $TOMCAT_HOME/webapps/ROOT.war",
```

Comment:
I think we should have a `build` command, a `deploy` command, and then separately a `build and deploy` command, which is this syntax.  I am not sure it's intuitive for users that build, automatically does a deploy - so we should provide all the variants.